log4j-mongodb/pom.xml

```diff
@@ -151,7 +150,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <skip>true</skip>
+          <skip>false</skip>
```

Comment:
You can remove the entire `<configuration>`, since `<skip>false` is the default.